packages/base/src/sync.ts

```diff
@@ -2,7 +2,7 @@ import { uuid } from './uuid'
 
 export interface SyncRequest {
   channels?: SyncChannels
-  webhooks?: SyncWebhook[]
+  webhooks?: Omit<SyncWebhook, 'token'>[]
```

Comment:
We will have to publish a new version of the base package before merging this PR!